### ISSUE TITLE
How to bypass chrome detection?

### ISSUE THREAD

**Issue by 4l4ddin** [Ignitetch/AdvPhishing#106]
Chrome detects the when entering the password for Paypal, is there a way to bypass that?

**Comment by deeshirazi** [comment]
did you find any solution to bypass Chrome? If you did please share

**Comment by 4l4ddin** [comment]
Yes I did, and also integrated results come in real time in telegram through a bot.
Could do a good price, telegram : Al4dd1

> did you find any solution to bypass Chrome? If you did please share

**Comment by deeshirazi** [comment]
> Yes I did, and also integrated results come in real time in telegram through a bot.
> Could do a good price, telegram : Al4dd1
> 
> > did you find any solution to bypass Chrome? If you did please share

If you did find out share with everybody here man!

**Comment by Hacker5820** [comment]
@deeshirazi @4l4ddin I do find a solution about that ;)
Use antibots, antidetect, blocker and anti url checker.

**Comment by im2gm** [comment]
Pls I need a help on a particular page dm when u are chance

**Comment by Hosted-EU** [comment]
There’s A New Phishing Tool Out There.
It Can Generate URLs For Phishing Pages.
And Able To Steal Cookies from Phishing Pages To Bypass 2FA 🔥

Watch Video On How It Works : https://youtube.com/watch?v=BJv3jFuJJOw
